temporalcli/commands.gen.go

```diff
@@ -244,7 +244,7 @@ type PayloadInputOptions struct {
 func (v *PayloadInputOptions) buildFlags(cctx *CommandContext, f *pflag.FlagSet) {
 	f.StringArrayVarP(&v.Input, "input", "i", nil, "Input value. Use JSON content or set --input-meta to override. Can't be combined with --input-file. Can be passed multiple times to pass multiple arguments.")
 	f.StringArrayVar(&v.InputFile, "input-file", nil, "A path or paths for input file(s). Use JSON content or set --input-meta to override. Can't be combined with --input. Can be passed multiple times to pass multiple arguments.")
-	f.StringArrayVar(&v.InputMeta, "input-meta", nil, "Input payload metadata as a `KEY=VALUE` pair. When the KEY is \"encoding\", this overrides the default (\"json/plain\"). Can be passed multiple times.")
+	f.StringArrayVar(&v.InputMeta, "input-meta", nil, "Input payload metadata as a `KEY=VALUE` pair. When the KEY is \"encoding\", this overrides the default (\"json/plain\"). Can be passed multiple times. Repeated metadata keys are applied to the corresponding inputs in the provided order.")
```

Comment:
```suggestion
	f.StringArrayVar(&v.InputMeta, "input-meta", nil, "Input payload metadata as a `KEY=VALUE` pair. When the KEY is \"encoding\", this overrides the default (\"json/plain\"). Can be passed multiple times. Passing the same key more than once will apply its values in sequence to the corresponding --input arguments.")
```

Honestly not sure this is better, but, I had no idea what was going on here until I saw the test. Maybe we just want an example in this help text?

Reply:
It's already pretty long so not sure we want to include the example in this single option's description.

Reply:
If we don't include the example, maybe still go with this wording change? I'm worried almost no one is going to understand this from just reading the helptext.

Luckily it probably won't come up much either, so I suppose there's that.